src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -475,10 +475,11 @@ public interface ITelegramBotClient
         /// <param name="chatId"><see cref="ChatId"/> for the target group</param>
         /// <param name="userId">Unique identifier of the target user</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <param name="untilDate">Date when the user will be unbanned, unix time. If user is banned for more than 366 days or less than 30 seconds from the current time they are considered to be banned forever</param>
         /// <returns><c>true</c> on success.</returns>
         /// <see href="https://core.telegram.org/bots/api#kickchatmember"/>
         Task<bool> KickChatMemberAsync(ChatId chatId, int userId,
-            CancellationToken cancellationToken = default(CancellationToken));
+            CancellationToken cancellationToken = default(CancellationToken), int untilDate = 0);
```

Comment:
- `untilDate` should be `DateTime` as indiated [here](https://github.com/MrRoundRobin/telegram.bot/pull/366#issuecomment-314290887)
- I always reordered the parameters so that the `cancellationToken` is the last. this caused a major change but I think this is ok for such a big change.